src/fmpz_poly/test/t-cyclotomic.c

```diff
@@ -57,7 +57,7 @@ TEST_FUNCTION_START(fmpz_poly_cyclotomic, state)
     fmpz_poly_t A, B;
     slong n;
 
-    for (n = 0; n <= 1000; n++)
+    for (n = 0; n <= FLINT_MIN(1000 * flint_test_multiplier(), 1000 + 100 * flint_test_multiplier()); n++)
```

Comment:
I have no idea why such a complex condition would be needed here.

Reply:
It could just go up to a fixed count of 200 or so. It doesn't really need to depend on ``FLINT_TEST_MULTIPLIER`` at all.

Reply:
I still think it is nice to reduce the time for the tests if one reduces the test multiplier, so I will set a limit.